### PR TITLE
Feature/tri 506

### PR DESCRIPTION
**Pull request by SaberIstos**

### CODE REVIEW

File: irs-api/src/main/resources/application.yml [hunk 1]
```diff
@@ -107,4 +107,4 @@ aspectTypes:
   default: SerialPartTypization
 
 digitalTwinRegistry:
-  url: https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com/semantics/twin-registry
+  url: https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html
```
Comment (ds-ext-kmassalski) [inline irs-api/src/main/resources/application.yml:110]: This is url to swagger, and not service

File: testing/IRS DEMO Collection.postman_collection.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
 							}
```
Comment (ds-ext-kmassalski) [inline testing/IRS DEMO Collection.postman_collection.json:21]: Can you also update postman environment files? (IRS_DEV.postman_environment.json, IRS_INT.postman_environment.json...)

**Comment by ds-ext-kmassalski** [comment]
Vault contains old values - it also has to be updated there

**Review by ds-ext-kmassalski — CHANGES_REQUESTED** [review]

File: irs-api/src/main/resources/application.yml (hunk 1 above)
Comment (SaberIstos) [reply irs-api/src/main/resources/application.yml:110]: you're right. I didn't cut the swagger url into: https://semantics.int.demo.catena-x.net/registry

File: testing/IRS DEMO Collection.postman_collection.json (hunk 2 above)
Comment (SaberIstos) [reply testing/IRS DEMO Collection.postman_collection.json:21]: I replaced the keycloak host url with: https://semantics.int.demo.catena-x.net

File: testing/IRS_DEV.postman_environment.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
 		},
```
Comment (ds-ext-kmassalski) [inline testing/IRS_DEV.postman_environment.json:9]: CLIENT_ID not updated

File: testing/IRS_INT.postman_environment.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
 		},
```
Comment (ds-ext-kmassalski) [inline testing/IRS_INT.postman_environment.json:9]: CLIENT_ID not updated

File: testing/IRS_INT.postman_environment.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
...
 		{
 			"key": "KEYCLOAK_HOST",
-			"value": "https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com",
+			"value": "https://semantics.int.demo.catena-x.net",
 			"enabled": true
 		},
 		{
```
Comment (ds-ext-kmassalski) [inline testing/IRS_INT.postman_environment.json:15]: IRS_LOCAL.postman_environment.json not updated

File: testing/IRS DEMO Collection.postman_collection.json [hunk 6]
```diff
@@ -178,7 +178,7 @@
...
 						],
 						"url": {
-							"raw": "https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com/semantics/twin-registry/lookup/shells?assetIds=[{\"key\":\"globalAssetId\",\"value\":\"urn:uuid:866b8c6f-8e59-4b9c-9e1d-61900bff91ce\"}]",
+							"raw": "https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html/lookup/shells?assetIds=[{\"key\":\"globalAssetId\",\"value\":\"urn:uuid:866b8c6f-8e59-4b9c-9e1d-61900bff91ce\"}]",
```
Comment (ds-ext-kmassalski) [inline testing/IRS DEMO Collection.postman_collection.json:181]: Wrong url!
"https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html"

File: testing/IRS DEMO Collection.postman_collection.json [hunk 7]
```diff
@@ -141,7 +141,7 @@
 							}
 						],
 						"url": {
-							"raw": "https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com/semantics/twin-registry/lookup/shells?assetIds=[{\"key\":\"VAN\",\"value\": \"OMA-TGFAYUHXFLHHUQQMPLTE\"}]",
+							"raw": "https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html/lookup/shells?assetIds=[{\"key\":\"VAN\",\"value\": \"OMA-TGFAYUHXFLHHUQQMPLTE\"}]",
```
Comment (ds-ext-kmassalski) [inline testing/IRS DEMO Collection.postman_collection.json:144]: Wrong url!
"https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html"

File: testing/IRS DEMO Collection.postman_collection.json [hunk 8]
```diff
@@ -104,7 +104,7 @@
 							}
 						],
 						"url": {
-							"raw": "https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com/semantics/twin-registry/lookup/shells?assetIds=[{\"key\": \"ManufacturerID\",\"value\":\"BPNL00000003B5MJ\"}]",
+							"raw": "https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html/lookup/shells?assetIds=[{\"key\": \"ManufacturerID\",\"value\":\"BPNL00000003B5MJ\"}]",
```
Comment (ds-ext-kmassalski) [inline testing/IRS DEMO Collection.postman_collection.json:107]: Wrong url!
"https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html"

File: testing/IRS DEMO Collection.postman_collection.json [hunk 9]
```diff
@@ -21,7 +21,7 @@
 							}
 						],
 						"url": {
-							"raw": "https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com/semantics/twin-registry/registry/shell-descriptors/:aasIdentifier",
+							"raw": "https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html/registry/shell-descriptors/:aasIdentifier",
```
Comment (ds-ext-kmassalski) [inline testing/IRS DEMO Collection.postman_collection.json:24]: Wrong url!
"https://semantics.int.demo.catena-x.net/registry/swagger-ui/index.html"

File: testing/IRS DEMO Collection.postman_collection.json (hunk 2 above)
Comment (ds-ext-kmassalski) [reply testing/IRS DEMO Collection.postman_collection.json:21]: Also step 5.0.0 [Authenticate with keycloak] requires update - remove "/iamcentralidp" from path

**Review by ds-ext-kmassalski — CHANGES_REQUESTED** [review]

File: testing/IRS_DEV.postman_environment.json (hunk 3 above)
Comment (SaberIstos) [reply testing/IRS_DEV.postman_environment.json:9]: Replaced with: sa-cl6-cx-9

File: testing/IRS_INT.postman_environment.json (hunk 4 above)
Comment (SaberIstos) [reply testing/IRS_INT.postman_environment.json:9]: Replaced with: sa-cl6-cx-9

File: testing/IRS_INT.postman_environment.json (hunk 5 above)
Comment (SaberIstos) [reply testing/IRS_INT.postman_environment.json:15]: {
			"key": "KEYCLOAK_HOST",
			"value": "https://semantics.int.demo.catena-x.net",
			"enabled": true
		},
		{
			"key": "CLIENT_ID",
			"value": "sa-cl6-cx-9",
			"enabled": true
		},

File: testing/IRS DEMO Collection.postman_collection.json (hunk 6 above)
Comment (SaberIstos) [reply testing/IRS DEMO Collection.postman_collection.json:181]: Changed to:
"url": {
	"raw": "https://semantics.int.demo.catena-x.net/registry/lookup/shells?assetIds [{\"key\":\"globalAssetId\",\"value\":\"urn:uuid:866b8c6f-8e59-4b9c-9e1d-61900bff91ce\"}]",

File: testing/IRS DEMO Collection.postman_collection.json (hunk 7 above)
Comment (SaberIstos) [reply testing/IRS DEMO Collection.postman_collection.json:144]: Changed to:
"url": {
	"raw": "https://semantics.int.demo.catena-x.net/registry/lookup/shells?assetIds [{\"key\":\"globalAssetId\",\"value\":\"urn:uuid:866b8c6f-8e59-4b9c-9e1d-61900bff91ce\"}]",

File: testing/IRS DEMO Collection.postman_collection.json (hunk 8 above)
Comment (SaberIstos) [reply testing/IRS DEMO Collection.postman_collection.json:107]: Changed to:
"url": {
	"raw": "https://semantics.int.demo.catena-x.net/registry/lookup/shells?assetIds [{\"key\":\"globalAssetId\",\"value\":\"urn:uuid:866b8c6f-8e59-4b9c-9e1d-61900bff91ce\"}]",

File: testing/IRS DEMO Collection.postman_collection.json (hunk 9 above)
Comment (SaberIstos) [reply testing/IRS DEMO Collection.postman_collection.json:24]: Changed to:
"url": {
	"raw": "https://semantics.int.demo.catena-x.net/registry/lookup/shells?assetIds [{\"key\":\"globalAssetId\",\"value\":\"urn:uuid:866b8c6f-8e59-4b9c-9e1d-61900bff91ce\"}]",

**Comment by SaberIstos** [comment]
5.0.0 [Authenticate with keycloak] in the json file IRS DEMO Collection.postman_collection.json:
Changed to:
				"url": {
					"raw": "{{KEYCLOAK_HOST}}/auth/realms/CX-Central/protocol/openid-connect/token",
					"host": [
						"{{KEYCLOAK_HOST}}"
					],
					"path": [
						"auth",
						"realms",
						"CX-Central",
						"protocol",
						"openid-connect",
						"token"
					]
				}
			},

**Review by ds-ext-kmassalski — APPROVED** [review]

**Pull request merged**